hazelcast/src/main/java/com/hazelcast/monitor/impl/LicenseInfo.java

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
Can you move this class to com.hazelcast.internal.management? 

This class doesn't belong to 'monitor'. This is reserved for the statistics implementations.